runtimes/cloudformation/cfnpatcher/fixtures/respect_ignores/opt_in_single_container.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Resources": {
+    "willpatch": {
+      "Type": "AWS::ECS::TaskDefinition",
+      "Properties": {
+        "RequiresCompatibilities": [
+          "FARGATE"
+        ],
+        "Tags": [
+          {
+            "Key": "kilt-include-containers",
+            "Value": "app"
+          },
+          {
+            "Key": "sometag",
+            "Value": "antani"
```

Comment:
Wonderful ahahah 💯

Reply:
I think there I typed it automatically :)